mod-fqm-manager/src/main/resources/corsair/mod-fqm-manager/features/query.feature

```diff
@@ -13,6 +13,7 @@ Feature: Query
     * def orgContactEntityTypeId = '7a7860cd-e939-504f-b51f-ed3e1e6b12b9'
```

Comment:
@okolawole-ebsco, at the top of this file (~10 before the start of the diff of this file), we've got a `callonce login testUser`. The equivalent line in your acquisitions PR (#1300) changes the `callonce` to `call`. Should we do the same thing here?

Reply:
It is only necessary if your scenario requires it. If multiple virtual users can execute the tests and everything looks green, no need to change it.